packages/connect/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@shipengine/connect",
-  "version": "1.12.0",
+  "version": "2.0.0",
```

```suggestion
  "version": "1.12.1",
```

Although this _is_ a breaking change in the api & runtime, it doesn't actually change the behavior from the perspective of the app developer, so we don't need a major bump here.

---

Updated to 1.12.2 (PR #124 updated to 1.12.1)